near-sdk/src/store/vec/mod.rs

```diff
@@ -253,6 +257,7 @@ where
     /// # Panics
     ///
     /// If `index` is out of bounds.
+    // TODO determine if this should be stabilized, included for backwards compat with old version
     pub fn replace(&mut self, index: u32, element: T) -> T {
```

Comment:
I'm surprised `Vec` doesn't have a `replace` function. It doesn't seem like there's any caveats to having such a thing, but forces people to do something like:
```rust
let old_val = std::mem::replace(&mut v[3], new_val);
```

Reply:
yeah, the convenience is why I left it, but the method isn't needed